lib/emailjs.js

```diff
@@ -208,6 +208,20 @@ exportable = (function () {
       setTimeout(retryErrorMails, retryInterval);  
     }
   }
+  
+  function emitWrapper () {
+    const eventName = arguments && arguments[0],
+      cb = arguments.length > 3 ? arguments[3] : null;
```

Comment:
dont use magic number 3

Reply:
no need to have additional check `arguments && arguments[0]` you can fetch the value with the trust of arguments being available in your function 
arguments[EV_NAME]

Reply:
collect callback = arguments[ERROR_CB] || "default to a function"?